### ISSUE TITLE
l_foucs: content allignment not stable/left

### ISSUE THREAD

**Issue by kreuzberger** [useblocks/sphinx-needs#743]
**Update**: Issue is reused as bug report. Bug description starts in https://github.com/useblocks/sphinx-needs/issues/743#issuecomment-1283639247

following the documentation i can add "custom" css files to change "default" needs styling done in css.
But if i add these css "normaly" in sphinx (e.g. with app.add_css_file) the generated html include order is
- default html style
- my styles
- sphinx-needs style

so i can not "overwrite" the default sphinx-needs styles because they are applied later.
is this intended?

E.g. my custom css is named "fsp.css" and added via add_css_file the generated html is

```
<link rel="stylesheet" type="text/css" href="_static/pygments.css" />
<link rel="stylesheet" type="text/css" href="_static/nature.css" />
<link rel="stylesheet" type="text/css" href="_static/fsp.css" />
<link rel="stylesheet" type="text/css" href="_static/assets/jsonview.bundle.css" />
<link rel="stylesheet" type="text/css" href="_static/sphinx-needs/libs/html/datatables.min.css" />
<link rel="stylesheet" type="text/css" href="_static/sphinx-needs/common.css" />
<link rel="stylesheet" type="text/css" href="_static/sphinx-needs/styles.css" />
```

E.g. i want to change default focus_l style back to "former" layout with ID written not vertical.

**Comment by danwos** [comment]
I'm currently not Infront of my pc to check the details, but does add_css_file have a second parameter for the order?

You could also use the way to add css via option html_css_files. Maybe these files get added later.

**Comment by kreuzberger** [comment]
You are right. The add_css_file has an priority argument. But the better way would be to use html_css_files. It takes the highest priority as default.
https://sphinxcontrib-needs.readthedocs.io/en/latest/layout_styles.html#own-css-file-on-sphinx-level should be adapted by either adding priority or the html_css_files configuration option.

BTW: If i use focus_l as layout. The content of the needs is not left aligned depending on the width of the table. The shorter the text, the more it is moved to the right. Which property is responsible for this behaviour?
![Screenshot_20221019_102316](https://user-images.githubusercontent.com/579125/196640763-c3479467-3fb1-4bf3-9562-c7407ee2c99e.png)

**Comment by danwos** [comment]
For the alignment problem: I don't have a clue and the behavior looks strange.
Maybe it also behaves different in a different browser.

I rename this issue, so we can keep it as bug report for the alignment problem.

**Comment by kreuzberger** [comment]
I am using firefox and brave browser mostly.

In `firefox` the requirement id is shown vertical, in `brave` browser it is shown horizontal. In both browsers the content seems "intended". But i am also using "nature" theme from sphinx (sphinx-internal theme). So question is what should be the reference "theme". So as i saw in your docs subdirectory, you are using "sphinx-immaterial" als html theme. 
What should a normal sphinx-user use as theme ( Maybe just the basic theme) to have a clear reference?

**Comment by danwos** [comment]
The goal is that Sphinx-Needs supports all themes. There is no reference theme.
But for sure, it get tested mostly with Sphinx-Immaterial and the Alabaster theme.
But some projects also use the ReadTheDocs or pydata theme.

And also the final layout shall be the same for all browsers.

So if something is not working in one theme, it's in most cases a bug in our CSS or the theme.
But also if it is a bug in theme, Sphinx-Needs normally tries to provide then some theme-specific CSS (mostly because the needed fixes are easier and get accepted quicker).

**Comment by kreuzberger** [comment]
I generated a demo project, the issue is quite simple to see. As you can see, but it was not mentioned before the requirements are loaded via needimport. Maybe this gives an additional hint/issue. 

[need_import.tar.gz](https://github.com/useblocks/sphinx-needs/files/9821974/need_import.tar.gz)

Below is  a screenshot from firefox

![image](https://user-images.githubusercontent.com/579125/196738988-1d10ba52-179a-4cba-bec6-a1453a0233eb.png)

Also interesting: The text is shifted to the right if the REQUIREMENT id is hovered by the mouse :smiley: 

And this one is from Brave:
![image](https://user-images.githubusercontent.com/579125/196739972-02e2274b-639d-4769-8b6d-896f28cf4682.png)
 here the wrapping of text is totally different too.
